@lingosta/app/components/Layout/MobileNav.tsx

```diff
@@ -6,15 +6,15 @@ import { BsTranslate, BsPencilFill } from 'react-icons/bs';
 import { TbDragDrop } from 'react-icons/tb';
 import { useUser } from '../../providers/UserProvider';
 
-const Footer = () => {
+const MobileNav = () => {
```

Comment:
Let's call this `BottomNav` since this can show up on web browsers on desktop as well if the width is small.